src/main/kotlin/no/nav/familie/ef/sak/mapper/SøknadsskjemaMapper.kt

```diff
@@ -131,7 +136,8 @@ object SøknadsskjemaMapper {
             }?.toSet() ?: emptySet()
 
     private fun tilDomene(bosituasjon: KontraktBosituasjon): Bosituasjon =
-            Bosituasjon(delerDuBolig = bosituasjon.delerDuBolig.verdi,
+            Bosituasjon(delerDuBolig = EnumTekstverdiMedSvarId(bosituasjon.delerDuBolig.verdi,
+                                                               bosituasjon.delerDuBolig.svarId ?: MANGLER_SVAR_ID),
```

Comment:
For meg er nullable bedre enn `MANGLER_SVAR_ID` iallefall

Reply:
Ja, mulig det er best å gjøre dem nullable og så flytte null-sjekken til frontend.